data/858/982/97/85898297.geojson

```diff
@@ -37,6 +37,9 @@
     "name:eng_x_preferred":[
         "Gro\u00dfer Biberhaufen"
     ],
+    "name:ger_x_preferred_x_variant":[
```

Comment:
Same as below, this language code should be `deu` not `ger`.

Reply:
Good catch.. the script has a iso_lookup table, but it's looking like my logic isn't respecting it.